paddle/fluid/inference/api/api.cc

```diff
@@ -112,6 +112,13 @@ void PaddleBuf::Free() {
   }
 }
 
+NativeConfig::NativeConfig() {
+  LOG(WARNING) << "The paddle::NativeConfig interface is deprecrated for now, "
```

Comment:
is going to be deprecated in the next release, plase use the latest paddle_infer::Config instead.